onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml

```diff
@@ -0,0 +1,18 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="PrincipalsWithoutAdminUpdate" tableDbType="NOT_IN_DB">
+                <columns>
+                    <column columnName="UserId">
+                        <columnTitle>User ID</columnTitle>
+                        <isKeyField>true</isKeyField>
+                    </column>
+                    column columnName="Type"/>
+                    column columnName="FirstName"/>
+                    column columnName="LastName"/>
```

Comment:
```suggestion
                    <column columnName="Type"/>
                    <column columnName="FirstName"/>
                    <column columnName="LastName"/>
```

Reply:
Looks like some typos here are preventing your query from validating

Reply:
Thank you Marty for finding the program errors for me.   I will ensure to use the "validating queries" tool within the schema browser the next time as Keith suggested.

Thanks,
Raymond